### PR TITLE
docs: include documentation for required parameters 

### PR DESCRIPTION
**Pull request by codyoss**
Currently it is unclear how to use some of the functions that
return calls. Their required parameters are undocumented. This
means a user of the library either needs to go look at the official
docs page for the api or find the corresponding do method in the
code and look for its private discovery json comment. This commit
starts to add parameter descriptions for required fields, those
that are passed to the Call method.

Fixes: #950

### CODE REVIEW

**Comment by codyoss** [comment]
@tbpg The first commit contains the actual code changes to the generator. LMK what you think of the formatting. I am open to suggestions!

File: accessapproval/v1/accessapproval-gen.go [hunk 1]
```diff
@@ -960,6 +964,11 @@ type FoldersUpdateAccessApprovalSettingsCall struct {
 // UpdateAccessApprovalSettings: Updates the settings associated with a
 // project, folder, or organization. Settings to update are determined
 // by the value of field_mask.
+//
+// - name: The resource name of the settings. Format is one of: *
+// "projects/{project}/accessApprovalSettings" *
+// "folders/{folder}/accessApprovalSettings" *
+// "organizations/{organization}/accessApprovalSettings"
```
Comment (tbpg) [inline accessapproval/v1/accessapproval-gen.go:971]: Can we make this show up properly?

File: abusiveexperiencereport/v1/abusiveexperiencereport-gen.go [hunk 2]
```diff
@@ -259,6 +259,9 @@ type SitesGetCall struct {
 }
 
 // Get: Gets a site's Abusive Experience Report summary.
+//
+// - name: The name of the site whose summary to get, e.g.
+// `sites/http%3A%2F%2Fwww.google.com%2F`. Format: `sites/{site}`
```
Comment (tbpg) [inline abusiveexperiencereport/v1/abusiveexperiencereport-gen.go:264]: Annoying question: is there a way to indent the second line?

Comment (tbpg) [inline abusiveexperiencereport/v1/abusiveexperiencereport-gen.go:264]: I wonder if we should add a period at the end if one isn't there?

**Review by tbpg — COMMENTED** [review]
Looks great!

File: abusiveexperiencereport/v1/abusiveexperiencereport-gen.go (hunk 2 above)
Comment (codyoss) [reply abusiveexperiencereport/v1/abusiveexperiencereport-gen.go:264]: That is a good idea!

**Review by tbpg — APPROVED** [review]
LGTM.

**Comment by kortschak** [comment]
Thanks

**Pull request merged**